backend/alembic/versions/e1e7929bebd8_initial_migration.py

```
@@ -0,0 +1,60 @@
+    sa.Column('owner_id', sqlmodel.sql.sqltypes.AutoString(), nullable=False),
+    sa.Column('title', sqlmodel.sql.sqltypes.AutoString(length=50), nullable=False),
+    sa.Column('description', sqlmodel.sql.sqltypes.AutoString(length=250), nullable=True),
+    sa.Column('category', sqlmodel.sql.sqltypes.AutoString(), nullable=True),
+    sa.Column('difficulty', sa.Integer(), nullable=True),
+    sa.Column('is_public', sa.Boolean(), nullable=False),
+    sa.Column('created_at', sa.DateTime(timezone=True), nullable=False),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_index(op.f('ix_quiz_category'), 'quiz', ['category'], unique=False)
+    op.create_table('users',
+    sa.Column('id', sa.Uuid(), nullable=False),
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)